app/src/test/java/org/feature/fox/coffee_counter/di/services/network/ApiServiceTest.kt

```diff
@@ -127,7 +129,7 @@ class ApiServiceTest {
             .setBody(generateStringBody("Created new Item successfully."))
         mockWebServer.enqueue(response)
 
-        val actualResponse = apiService.postItem(body)
+        val actualResponse = apiService.postItem(bearerToken, body)
```

Comment:
The test cases should also assert if the authorization header was send correctly to mock webserver